charts/cosi/values.yaml

```diff
@@ -23,15 +23,15 @@ provisioner:
   logLevel: 4
   # logFormat is the logging format for the COSI driver provisioner,
   # Possible values: "json" "text".
-  logFormat: "text"
+  logFormat: "json"
   # otelEndpoint specifies the endpoint on which the OTEL Collector is set up and to which data is sent over gRPC.
-  otelEndpoint: "otel-collector.namespace:4317"
+  otelEndpoint: ""
   # image specifies the COSI driver provisioner container image.
   image:
     # repository is the COSI driver provisioner container image repository.
-    repository: "docker.io/dellemc/cosi"
+    repository: "quay.io/dell/container-storage-modules/cosi"
```

Comment:
Instead of using repository and tag, can we do `image: quay.io/dell/container-storage-modules/cosi: v1.0.0`? This aligns with our other values file and allows updates from dependabot.